scipy/stats/tests/test_stats.py

```diff
@@ -3998,5 +3998,48 @@ def test_weighted_stouffer2(self):
                                      weights=np.array((1, 4, 9)))
         assert_approx_equal(p, 0.1464, significant=4)
 
+def test_jensen_shannon_divergence():
+    for _ in range(8):
+        a = np.random.random(16)
+        b = np.random.random(16)
+        c = a+b
+
+        assert_(stats.jensen_shannon_divergence(a, a) < 1e-4)
```

Comment:
I think for pytest float comparisons a plain `assert` is just fine now? This was written many years ago I think, and is perhaps even now still not so bad.